cli/ioctl/cmd/config/config.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package config
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+var configFileName string
+
+// ConfigCmd represents the config command
+var ConfigCmd = &cobra.Command{
+	Use:       "config",
+	Short:     "Set or get configuration for ioctl client",
+	ValidArgs: []string{"set", "get"},
+	Args:      cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("Print: " + strings.Join(args, " "))
+	},
+}
+
+func init() {
+	dir := os.Getenv("HOME") + "/.config/iotex"
```

Comment:
store here: ~/.config/ioctl/config_default, also make sure this works on both macos and ubuntu/linux

Reply:
yes, Getenv("HOME") is for both mac and Linux